indico/htdocs/sass/modules/_event_management.scss

```diff
@@ -519,3 +519,43 @@ th.i-table {
         }
     }
 }
+
+#track-list {
+    list-style-type: none;
+    padding-left: 0;
+    width: 800px;
+
+    .i-box {
+        padding: 10px 10px 0 10px ;
+        margin-bottom: 10px;
```

Comment:
You don't need to specify `margin-bottom` here. Simply wrap the `.i-box` elements in a `div.i-box-group.vert` and that will take care of spacing them conveniently.